crates/arroyo-connectors/src/kafka/mod.rs

```diff
@@ -926,6 +927,14 @@ pub fn client_configs(
         }
     };
 
+    for prop in connection.connection_properties.iter() {
+        if let Some((k, v)) = prop.split_once('=') {
+            client_configs.insert(k.to_string(), v.to_string());
+        } else {
+            bail!("invalid connection property: {}", prop);
+        }
+    }
+
```

Comment:
With this change we'll be able to set client configs from both the connection and the table. I think it would be good to warn when iterating through the table configs (on 939) if both define the same property.